composer.lock

```diff
@@ -3537,33 +3537,29 @@
         },
         {
             "name": "paragonie/random_compat",
-            "version": "v2.0.17",
```

Comment:
????

Reply:
https://github.com/vektorprogrammet/vektorprogrammet/pull/909/files/745f47bf9e86a01cb8ca5137e453c6e132398475#diff-1da2c7edc898c70e5a79a9997c98ceccR3540

Reply:
Hva gjør den pluginen for noe? Er det bare noe som trengs hvis vi kjører php 5.x? 
https://github.com/paragonie/random_compat#version-99999

Reply:
Den kommer nok indirekte fra en annen pakke vi har installert. Kan sikkert bare ignorere den